app.rb

```diff
@@ -6,16 +6,18 @@
 require_relative './classes/label'
 require_relative './classes/music_album'
 require_relative './classes/genre'
+require_relative './classes/game'
+require_relative './classes/author'
 
 class App
   def initialize
-    # @items = []
+    @items = []
     # @genres = []
     @authors = []
     # @labels = []
     # @sources = []
 
-    @items = LoadData.load_items || []
+    # @items = LoadData.load_items || []
```

Comment:
- [ ] Kindly, modify the line that should be commented, so the app restores the Items information from the JSON file when started, and @items variable is not empty